### PR TITLE
Create GitHub widget to list all workflows

### PR DESCRIPTION
**Pull request by ebarriosjr**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by ebarriosjr** [comment]
I have another PR ready to go for adding this to the catalog plugin in the CI/CD tab once this one is merged to master.

File: plugins/github-actions/src/components/Widget/Widget.tsx [hunk 1]
```diff
@@ -108,3 +108,54 @@ export const Widget = ({
...
 };
+
+const WidgetListContent = ({
+  error,
+  loading,
+  branch,
+}: {
+  error?: Error;
+  loading?: boolean;
+  lastRun: WorkflowRun;
```
Comment (freben) [inline plugins/github-actions/src/components/Widget/Widget.tsx:119]: aren't you going to pass this into the table or so?

File: plugins/github-actions/src/components/Widget/Widget.tsx [hunk 2]
```diff
@@ -108,3 +108,54 @@ export const Widget = ({
...
+  branch: string;
+}) => {
+  if (error) return <Typography>Couldn't fetch {branch} runs</Typography>;
+  if (loading) return <LinearProgress />;
+  return <WorkflowRunsTable />;
+};
+
+export const WidgetList = ({
```
Comment (freben) [inline plugins/github-actions/src/components/Widget/Widget.tsx:127]: maybe name it more descriptively along the lines of `RecentWorkflowRunsCard` - that represents what it is and what it does better, I think

**Review by freben — APPROVED** [review]

File: plugins/github-actions/src/components/Widget/Widget.tsx (hunk 2 above)
Comment (ebarriosjr) [reply plugins/github-actions/src/components/Widget/Widget.tsx:127]: Maybe `WidgetRecentWorkflowRunsCard`. What do you think of this name @freben ?

Comment (freben) [reply plugins/github-actions/src/components/Widget/Widget.tsx:127]: If you absolutely want the word Widget in there, sure :) I was thinking that a card is a card, so to speak.

Comment (ebarriosjr) [reply plugins/github-actions/src/components/Widget/Widget.tsx:127]: Ok, I think having the work widget in it make the purpose more clear.

Comment (ebarriosjr) [reply plugins/github-actions/src/components/Widget/Widget.tsx]: The name is to long with Widget on it. So I am dropping the widget :P

File: plugins/github-actions/src/components/Widget/Widget.tsx (hunk 1 above)
Comment (ebarriosjr) [reply plugins/github-actions/src/components/Widget/Widget.tsx]: removed lastRun parameter from the function in the next commit.

**Review by freben — APPROVED** [review]

**Pull request merged**